### PR TITLE
Fixes hystrix timeout

### PR DESCRIPTION
**Pull request by ukriish**
- Fixes issue where hystrix timeout was being set in nanosecond instead
of millisecond

### CODE REVIEW

**Comment by ukriish** [comment]
#57

File: hystrix/hystrix_client.go [hunk 1]
```diff
@@ -67,7 +67,7 @@ func NewClient(opts ...Option) *Client {
 	}
 
 	hystrix.ConfigureCommand(client.hystrixCommandName, hystrix.CommandConfig{
-		Timeout:                int(client.hystrixTimeout),
+		Timeout:                int(client.hystrixTimeout / 1000000),
```
Comment (rShetty) [inline hystrix/hystrix_client.go]: Is there a way to add a test and catch this ?

Comment (ukriish) [reply hystrix/hystrix_client.go]: @rShetty yea, can add basic test for this.

Comment (ukriish) [reply hystrix/hystrix_client.go]: The test would mostly be that 10*second would give a value of 10000 millisecond. Don't think there is any other way to test this.

Thoughts?

Comment (ukriish) [reply hystrix/hystrix_client.go]: also might be good to add a size cap; since int conversion of huge number might throw overflow exception.

Comment (rShetty) [reply hystrix/hystrix_client.go]: Yes, size cap is something which we can think about later. But a basic test and this PR looks good.

Comment (ukriish) [reply hystrix/hystrix_client.go]: @rShetty Have added tests

**Pull request merged**